### PR TITLE
octoprint.python.pkgs.octoprint-dashboard: init at 1.13.0

### PR DESCRIPTION
**Pull request by j0hax**
###### Motivation for this change
Added the plugin `Octoprint-Dashboard` and its optional dependency `Octoprint-DisplayLayerProgress`

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3623,6 +3623,12 @@
     githubId = 26877687;
     name = "Yurii Izorkin";
   };
+  j0hax = {
+    name = "Johannes Arnold";
+    email = "johannes.arnold@stud.uni-hannover.de";
+    github = "j0hax";
+    githubId = 3802620;
+  };
```
Comment (trepetti) [inline maintainers/maintainer-list.nix:3631]: This should be its own commit:

`maintainers: add j0hax`

File: pkgs/applications/misc/octoprint/plugins.nix [hunk 2]
```diff
@@ -252,4 +252,43 @@ in {
...
+      sha256 = "0yv8gy5dq0rl7zxkvqa98az391aiixl8wbzkyvbmpjar9r6whdzm";
+    };
+
+    meta = with stdenv.lib; {
+      description = "OctoPrint-Plugin that sends the current progress of a print via M117 command";
+      homepage = "https://github.com/OllisGit/OctoPrint-DisplayLayerProgress";
+      license = licenses.agpl3;
+      maintainers = with maintainers; [ j0hax ];
+    };
+  };
```
Comment (trepetti) [inline pkgs/applications/misc/octoprint/plugins.nix:273]: This should be its own commit:

`octoprint.python.pkgs.displaylayerprogress: init at 1.23.2`

File: pkgs/applications/misc/octoprint/plugins.nix [hunk 3]
```diff
@@ -252,4 +252,43 @@ in {
...
+
```
Comment (trepetti) [inline pkgs/applications/misc/octoprint/plugins.nix:274]: This line of whitespace can be removed for consistency.

File: pkgs/applications/misc/octoprint/plugins.nix [hunk 4]
```diff
@@ -252,4 +252,43 @@ in {
...
+      sha256 = "1879l05gkkryvhxkmhr3xvd10d4m7i0cr3jk1gdcv47xwyr6q9pf";
+    };
+
+    meta = with stdenv.lib; {
+      description = "A dashboard for Octoprint";
+      homepage = "https://github.com/StefanCohen/OctoPrint-Dashboard";
+      license = licenses.agpl3;
+      maintainers = with maintainers; [ j0hax ];
+    };
+  };
```
Comment (trepetti) [inline pkgs/applications/misc/octoprint/plugins.nix:293]: This should be its own commit:

`octoprint.python.pkgs.octoprint-dashboard: init at 1.13.0`

**Review by trepetti — COMMENTED** [review]
Overall looks good, but it should be split into three commits with rebase/force-push. I realize this is a pain, but I think it will make it easier to get merged since they take the commit log seriously (for good reason).

**Comment by trepetti** [comment]
@ofborg eval

**Comment by trepetti** [comment]
Not sure what this error is and wanna see in more detail what is going wrong.

**Comment by j0hax** [comment]
Strange, must have been a one-time error.

**Review by trepetti — APPROVED** [review]
LGTM
